samples/nextjs/src/lib/tracking-service.ts

```diff
@@ -1,7 +1,11 @@
-import { TrackingService } from '@sitecore-jss/sitecore-jss-tracking';
+import { DummyTrackingService, TrackingService } from '@sitecore-jss/sitecore-jss-tracking';
 import config from 'temp/config';
 
-export const trackingService = new TrackingService({
-  endpoint: config.trackingEndpoint,
-  siteName: config.jssAppName,
-});
+export const trackingService =
+  config.isTrackingEnabled.toLocaleLowerCase() === 'true'
```

Comment:
It should be `boolean` type, not a string. Please, look, at this `isTrackingEnabled` variable

Reply:
Config generation does not support boolean variables.

Reply:
Got it, we can leave it for now